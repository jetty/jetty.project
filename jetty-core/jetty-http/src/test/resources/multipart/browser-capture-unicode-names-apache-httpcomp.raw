--yRxfRSltW63lJPc9oHOOVyCn-SmDG6i4Ts9M4E6
Content-Disposition: form-data; name="こんにちは世界"
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Greetings 1
--yRxfRSltW63lJPc9oHOOVyCn-SmDG6i4Ts9M4E6
Content-Disposition: form-data; name="%E3%81%93%E3%82%93%E3%81%AB%E3%81%A1%E3%81%AF%E4%B8%96%E7%95%8C"
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Greetings 2
--yRxfRSltW63lJPc9oHOOVyCn-SmDG6i4Ts9M4E6--
