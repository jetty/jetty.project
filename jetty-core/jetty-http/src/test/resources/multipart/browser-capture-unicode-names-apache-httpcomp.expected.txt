Request-Header|Accept-Encoding|gzip,deflate
Request-Header|Connection|keep-alive
Request-Header|Content-Length|475
Request-Header|Content-Type|multipart/form-data; boundary=yRxfRSltW63lJPc9oHOOVyCn-SmDG6i4Ts9M4E6
Request-Header|Host|localhost:9090
Request-Header|User-Agent|Apache-HttpClient/4.5.5 (Java/1.8.0_162)
Request-Header|X-BrowserId|apache-httpcomp
Parts-Count|2
Part-ContainsContents|こんにちは世界|Greetings 1
Part-ContainsContents|%E3%81%93%E3%82%93%E3%81%AB%E3%81%A1%E3%81%AF%E4%B8%96%E7%95%8C|Greetings 2

